applications/nrf_desktop/description.rst

```diff
@@ -382,7 +382,7 @@ The nRF Desktop application does not use a single :file:`prj.conf` file.
 Before you start testing the application, you can select one of the build types supported by the application.
 Not every board supports all of the mentioned build types.
 
-See :ref:`app_build_additions_build_types` and :ref:`modifying_build_types` for more information about this feature of the |NCS|.
+See :ref:`app_build_additions_build_types` and :ref:`cmake_options` for more information.
```

Comment:
Could we explicitly advise not to switch to `FILE_SUFFIX` yet to prevent confusion (even though `CONF_FILE` is deprecated)? Using `FILE_SUFFIX` would result in "silent failures" -> build would be completed, but used configuration would not be proper.

Reply:
Updated the notes in both conceptual sections in config_and_build_system.rst.